### ISSUE TITLE
Minecraft Bedrock Support

### ISSUE THREAD

**Issue by Hamza102955** [CraftingStore/Suggestions#173]
**Is your feature request related to a problem? Please describe.**
No. My request is to implement support/ for Minecraft Bedrock. This is because Minecraft Bedrock is limited to just Tebex for E-Commerce, and Tebex is extremely expensive. I looked around, and saw that Craftingstore has cheap plans and a 30-day trial with my host. I believe it would be a win/win for both of us as Minecraft Bedrock is available on multiple platforms, Windows 10, XBOX One and IOS / Android.

**Describe the solution you'd like**
You could port the plugin to Minecraft Bedrock. The most common server software is Pocketmine, so I would suggest making a Pocketmine plugin- it is in .php. This is the same step Tebex took and if you do this, I would definitely use this for my server with the paid plans, and recommend it to all my friends

**Additional context**

**Comment by GMatrixGames** [comment]
Although this hasn't had any replies, I'm working on a port to Nukkit which is a Java Bedrock Server implementation.

**Comment by Tim-NL** [comment]
Hi,

We've tested Pocketmine-MP and it seems to support RCON just fine, we've tested it, and CraftingStore is able to execute commands on here out-of-the-box. 

We've described how to set-up RCON on PocketMine-MP here: https://help.craftingstore.net/books/plugin-help/page/how-to-install-the-plugin#bkmrk-minecraft%3A-bedrock-%2F

We've also added "Minecraft: Bedrock" as a selectable game type for new stores, this will hide all other plugins that we offer, and just show RCON. 

We might create a dedicated plugin in the future, so there can be a /buy menu, but at this point in time, we'll limit it to just command processing (the main focus of our platform). 

-Tim
